perun-core/src/main/java/cz/metacentrum/perun/core/provisioning/IdHasher.java

```diff
@@ -0,0 +1,64 @@
+package cz.metacentrum.perun.core.provisioning;
+
+import cz.metacentrum.perun.core.api.Facility;
+import cz.metacentrum.perun.core.api.Group;
+import cz.metacentrum.perun.core.api.Member;
+import cz.metacentrum.perun.core.api.Resource;
+import cz.metacentrum.perun.core.api.User;
+import cz.metacentrum.perun.core.api.Vo;
+
+/**
+ * @author Vojtech Sassmann <vojtech.sassmann@gmail.com>
+ */
+public class IdHasher implements Hasher {
```

Comment:
Please add at least a simple Javadoc about this class.

Reply:
Done.